docs/architecture/geodata-pipeline-arch-geoplatform.mmd

```diff
@@ -0,0 +1,28 @@
+graph TD
+    Dataset1["Dataset 1"]-->Score
+    Dataset2["Dataset 2"]-->Score
+    Census["Census TIGER Data"]-->CGTiger
+    subgraph "Owned by Geoplatform"
+        CGTiger["Create GeoJSON from Shapefile with osgeo/gdal"]-->TS3
+        
+    end
+    TS3("TIGER GeoJSON (S3)")-->CGJ
+    Score["Create Score CSV"]--Event Notification-->CSV
+    CSV("CSV (S3)")--"Event Notification (Geoplatform)"-->CGJ
+    subgraph "Owned by Geoplatform"
+        CGJ["Combine (ogr2ogr)+ Create GeoJSON"]--Event Notification-->GeoJSON
```

Comment:
@switzersc-usds -- is this part all automated? Or do they need to manually run some of this?

Reply:
Automated!